lib/tasks/import.rake

```diff
@@ -28,4 +28,47 @@ namespace :import do
     puts "Missing ETDs: #{missing_etds.inspect}" if missing_etds.present?
     puts "Import Complete #{Time.zone.now}"
   end
+
+  task :adrl_arks, [:file_path] => :environment do |_t, args|
+    puts "Beginning import #{Time.zone.now}"
+    # CSV file contents would be
+    # merritt,proquest
+    # ark:/13030/m00999g9,ProQuestID:0035D
+    csv = CSV.read(args[:file_path], headers: true)
+    count = 0
+    missing_files = non_adrl = []
+
+    # rewrite CSV with each run
+    # using 'w' mode
+    CSV.open("tmp/adrl_arks.csv",
+             "w",
+             write_headers: true,
+             headers: %w[proquest merritt ucsb]) do |row|
+      ETD.find_each do |etd|
+        next if etd.merritt_id.present?
+
+        pq_name = etd.file_sets.first.original_file.file_name.first
+        if pq_name.blank?
+          missing_files << etd.id
```

Comment:
this gets hit if the original_file has an empty name. is this something you saw happinging in development? staging? do we expect this set to be empty on a healthy run?